runtime/wasm/src/host_exports.rs

```diff
@@ -129,13 +130,30 @@ impl HostExports {
         )))
     }
 
+    /// Some data sources have indeterminism or different notions of time. These
+    /// need to be each be stored separately to separate causality between them,
+    /// and merge the results later. Right now, this is just ethereum but it
+    /// will be expanded for ipfs and the availability chain.
+    fn poi_namespace(&self) -> &str {
+        "ethereum"
```

Comment:
Strictly speaking, this should be the concrete network, not just the family. For now, it doesn't matter since we don't ever have subgraphs that straddle networks, so that for a mainnet subgraph `ethereum` means `mainnet` and for a Rinkeby subgraph it means `rinkeby`, but it would be cleaner to actually use the network.

Reply:
Yeah, agreed